src/utils/cookie.ts

```diff
@@ -0,0 +1,29 @@
+import { jwtDecode } from 'jwt-decode';
+
+export const getCookie = (name: string) => {
```

Comment:
You can give a union type for name which could be used on all the function which handles cookie.

```javascript
 type cookieName = "accessToken" | "refreshToken"
 
 name: cookieName
```
Currently, these are the only values we need, so giving type like this provide suggestion while calling the function.